src/main/java/org/opensearch/sdk/BaseExtensionRestHandler.java

```diff
@@ -127,4 +134,58 @@ protected ExtensionRestResponse createJsonResponse(RestRequest request, RestStat
             return new ExtensionRestResponse(request, status, fieldName + ": " + responseStr);
         }
     }
+
+    /**
+     * Returns a String message of the detail of any unrecognized error occurred
+     *
+     * @param request The request that caused the exception
+     * @param invalids invalid strings
+     * @param candidates candidates involved
```

Comment:
Are candidates "involved"?  I don't know what this means. Looking at the code it appears to be using levenshtein distance to provide a "did you mean" functionality.  So say so here.  "A set of words that are most likely to be the valid strings determined invalid, to be suggested to the user."

Reply:
Updated